dask/array/slicing.py

```diff
@@ -445,14 +481,9 @@ def issorted(seq):
     >>> issorted([3, 1, 2])
     False
     """
-    if not seq:
+    if len(seq) == 0:
         return True
-    x = seq[0]
-    for elem in seq[1:]:
-        if elem < x:
-            return False
-        x = elem
-    return True
+    return np.all(sorted(seq) == seq)
```

Comment:
The reason the original code didn't want to use sort() or sorted() is probably that those functions can have O(n log n) behaviour or worse.